quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testExportHelp.unix.approved.txt

```diff
@@ -131,11 +131,13 @@ Logging:
 Export:
 
 --dir <dir>          Set the path to a directory where files will be created with the exported data.
---file <file>        Set the path to a file that will be created with the exported data.
+--file <file>        Set the path to a file that will be created with the exported data. To export
+                       more than 500 users, export to a directory with different files instead.
```

Comment:
How did you come up with the `500` number? Does it depend on the associations like roles, groups, attributes, etc?

Reply:
I've taken this from the old `importExport.adoc` documentation which I remove in this PR (it was a file which wasn't used any more).

Looking at the linked bug report, this number (and also the current default of 50) still seem to be valid.

Reply:
My point is that the number is too generic and the explicit support for 500 and below users might be impacted by how much data you hold on your users.

I can have a hand full of users and thousands of roles, groups, attributes, etc. Which should also impact the export, right?

Perhaps, we can just say that "Depending on how many users and how much data they have, the best is to use separate files". Something along these lines.

Reply:
When they have thousands of roles and groups, they will have an exponentially slowed down export anyway, and using the single files for the users won't help. I'd rather have something more actionable like "500 users", and it has been line this in the docs before.